### ISSUE TITLE
Warning: CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+

### ISSUE THREAD

**Issue by kravciak** [rancher/rancher#31514]
Rancher: master-head (465bb3b)
Monitoring: 9.4.203-rc04
Istio: 1.8.300-rc01
Kiali: 1.29.000-rc00
Cluster: Amazon EC2 v1.20.4

To reproduce (all settings left default):
 - go to Cluster Explorer -> Apps & Marketplace
 - install `Monitoring`
 - install `Istio`
 - go to installed apps
 - remove istio & kiali

```
helm upgrade --install=true --namespace=istio-system --timeout=10m0s --values=/home/shell/helm/values-rancher-kiali-server-crd-1.29.000-rc00.yaml --version=1.29.000-rc00 --wait=true rancher-kiali-server-crd /home/shell/helm/rancher-kiali-server-crd-1.29.000-rc00.tgz
Release "rancher-kiali-server-crd" does not exist. Installing it now.
W0226 13:33:37.921949 19 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
creating 1 resource(s)
W0226 13:33:37.938888 19 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
beginning wait for 1 resources with timeout of 10m0s
W0226 13:33:39.952490 19 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
NAME: rancher-kiali-server-crd
LAST DEPLOYED: Fri Feb 26 13:33:37 2021
NAMESPACE: istio-system
STATUS: deployed
REVISION: 1
TEST SUITE: None
---------------------------------------------------------------------
SUCCESS: helm upgrade --install=true --namespace=istio-system --timeout=10m0s --values=/home/shell/helm/values-rancher-kiali-server-crd-1.29.000-rc00.yaml --version=1.29.000-rc00 --wait=true rancher-kiali-server-crd /home/shell/helm/rancher-kiali-server-crd-1.29.000-rc00.tgz


helm uninstall --namespace=istio-system rancher-kiali-server-crd
Starting delete for "monitoringdashboards.monitoring.kiali.io" CustomResourceDefinition
W0226 12:40:36.041919 19 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
release "rancher-kiali-server-crd" uninstalled
---------------------------------------------------------------------
SUCCESS: helm uninstall --namespace=istio-system rancher-kiali-server-crd
```

**Comment by kravciak** [comment]
I see various warnings about deprecations also in v2.5-head (5a5c039) for multiple charts, for example:

```
Monitoring:
rbac.authorization.k8s.io/v1beta1 ClusterRole is deprecated in v1.17+, unavailable in v1.22+; use rbac.authorization.k8s.io/v1 ClusterRole
rbac.authorization.k8s.io/v1beta1 ClusterRoleBinding is deprecated in v1.17+, unavailable in v1.22+; use rbac.authorization.k8s.io/v1 ClusterRoleBinding
rbac.authorization.k8s.io/v1beta1 Role is deprecated in v1.17+, unavailable in v1.22+; use rbac.authorization.k8s.io/v1 Role
rbac.authorization.k8s.io/v1beta1 RoleBinding is deprecated in v1.17+, unavailable in v1.22+; use rbac.authorization.k8s.io/v1 RoleBinding
apiregistration.k8s.io/v1beta1 APIService is deprecated in v1.19+, unavailable in v1.22+; use apiregistration.k8s.io/v1 APIService
admissionregistration.k8s.io/v1beta1 MutatingWebhookConfiguration is deprecated in v1.16+, unavailable in v1.22+; use admissionregistration.k8s.io/v1 MutatingWebhookConfiguration
admissionregistration.k8s.io/v1beta1 ValidatingWebhookConfiguration is deprecated in v1.16+, unavailable in v1.22+; use admissionregistration.k8s.io/v1 ValidatingWebhookConfiguration

CIS:
apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
rbac.authorization.k8s.io/v1beta1 ClusterRole is deprecated in v1.17+, unavailable in v1.22+; use rbac.authorization.k8s.io/v1 ClusterRole
rbac.authorization.k8s.io/v1beta1 ClusterRoleBinding is deprecated in v1.17+, unavailable in v1.22+; use rbac.authorization.k8s.io/v1 ClusterRoleBinding

OPA:
W0301 14:22:34.601696 36 warnings.go:67] admissionregistration.k8s.io/v1beta1 ValidatingWebhookConfiguration is deprecated in v1.16+, unavailable in v1.22+; use admissionregistration.k8s.io/v1 ValidatingWebhookConfiguration

Logging:
W0301 14:36:13.336454 18 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
```

This is probably caused by new Kubernetes Version: v1.20.4, not rancher version.

**Comment by aiyengar2** [comment]
The Monitoring-related concerns from this ticket should be resolved once https://github.com/rancher/rancher/issues/29871 is complete.

**Comment by brendarearden** [comment]
Kiali has already fix this deprecation warning and will resolve once we upgrade our kiali version to match upstream. I will be tracking the kiali upgrade in a different issue that I have linked.

**Comment by thedadams** [comment]
The fix is verified on `master-head`. The logs from uninstalling istio version 1.9.300-rc01 do not produce the deprecation warning:
```helm uninstall --namespace=istio-system rancher-istio
creating 1 resource(s)
Watching for changes to Job istioctl-uninstaller with timeout of 5m0s
Add/Modify event for istioctl-uninstaller: MODIFIED
istioctl-uninstaller: Jobs active: 1, jobs failed: 0, jobs succeeded: 0
Add/Modify event for istioctl-uninstaller: MODIFIED
Starting delete for "istioctl-uninstaller" Job
Starting delete for "kiali" Service
Starting delete for "kiali" Deployment
Starting delete for "kiali-controlplane" RoleBinding
Starting delete for "kiali-controlplane" Role
Starting delete for "istio-installer" ClusterRoleBinding
Starting delete for "kiali" ClusterRoleBinding
Starting delete for "istio-edit" ClusterRole
Starting delete for "istio-view" ClusterRole
Starting delete for "kiali-viewer" ClusterRole
Starting delete for "kiali" ClusterRole
Starting delete for "istio-admin" ClusterRole
Starting delete for "istio-installer" ClusterRole
Starting delete for "istio-installer-base" ConfigMap
Starting delete for "kiali" ConfigMap
Starting delete for "kiali-console" ConfigMap
Starting delete for "istio-installer" ServiceAccount
Starting delete for "kiali" ServiceAccount
Starting delete for "vertx-server" MonitoringDashboard
Starting delete for "envoy" MonitoringDashboard
Starting delete for "go" MonitoringDashboard
Starting delete for "kiali" MonitoringDashboard
Starting delete for "micrometer-1.0.6-jvm-pool" MonitoringDashboard
Starting delete for "micrometer-1.0.6-jvm" MonitoringDashboard
Starting delete for "micrometer-1.1-jvm" MonitoringDashboard
Starting delete for "microprofile-1.1" MonitoringDashboard
Starting delete for "microprofile-x.y" MonitoringDashboard
Starting delete for "nodejs" MonitoringDashboard
Starting delete for "quarkus" MonitoringDashboard
Starting delete for "springboot-jvm-pool" MonitoringDashboard
Starting delete for "springboot-jvm" MonitoringDashboard
Starting delete for "springboot-tomcat" MonitoringDashboard
Starting delete for "thorntail" MonitoringDashboard
Starting delete for "tomcat" MonitoringDashboard
Starting delete for "vertx-client" MonitoringDashboard
Starting delete for "vertx-eventbus" MonitoringDashboard
Starting delete for "vertx-jvm" MonitoringDashboard
Starting delete for "vertx-pool" MonitoringDashboard
Starting delete for "istio-component-monitor" ServiceMonitor
Starting delete for "envoy-stats-monitor" ServiceMonitor
release "rancher-istio" uninstalled

---------------------------------------------------------------------
SUCCESS: helm uninstall --namespace=istio-system rancher-istio
---------------------------------------------------------------------
```

**Comment by sowmyav27** [comment]
Pending validations in Monitoring v2.
Pending fixes in CIS, OPA, Logging v2 charts

**Comment by thedadams** [comment]
The fix is verified on `master-head` for rancher-monitoring. The logs from uninstalling version 14.5.100-rc7 did no produce the depreciation warning:
```
helm uninstall --namespace=cattle-monitoring-system rancher-monitoring
Starting delete for "v1beta1.custom.metrics.k8s.io" APIService
Starting delete for "rancher-monitoring-ingress-nginx" Service
Starting delete for "rancher-monitoring-grafana" Service
Starting delete for "rancher-monitoring-kube-state-metrics" Service
Starting delete for "rancher-monitoring-prometheus-adapter" Service
Starting delete for "rancher-monitoring-prometheus-node-exporter" Service
Starting delete for "pushprox-kube-controller-manager-proxy" Service
Starting delete for "pushprox-kube-controller-manager-client" Service
Starting delete for "pushprox-kube-etcd-proxy" Service
Starting delete for "pushprox-kube-etcd-client" Service
Starting delete for "pushprox-kube-proxy-proxy" Service
Starting delete for "pushprox-kube-proxy-client" Service
Starting delete for "pushprox-kube-scheduler-proxy" Service
Starting delete for "pushprox-kube-scheduler-client" Service
Starting delete for "rancher-monitoring-alertmanager" Service
Starting delete for "rancher-monitoring-coredns" Service
Starting delete for "rancher-monitoring-operator" Service
Starting delete for "rancher-monitoring-prometheus" Service
Starting delete for "rancher-monitoring-operator" Deployment
Starting delete for "rancher-monitoring-grafana" Deployment
Starting delete for "rancher-monitoring-kube-state-metrics" Deployment
Starting delete for "rancher-monitoring-prometheus-adapter" Deployment
Starting delete for "pushprox-kube-controller-manager-proxy" Deployment
Starting delete for "pushprox-kube-etcd-proxy" Deployment
Starting delete for "pushprox-kube-proxy-proxy" Deployment
Starting delete for "pushprox-kube-scheduler-proxy" Deployment
Starting delete for "pushprox-kube-scheduler-client" DaemonSet
Starting delete for "pushprox-kube-controller-manager-client" DaemonSet
Starting delete for "pushprox-kube-etcd-client" DaemonSet
Starting delete for "pushprox-kube-proxy-client" DaemonSet
Starting delete for "rancher-monitoring-prometheus-node-exporter" DaemonSet
Starting delete for "rancher-monitoring-alertmanager" RoleBinding
Starting delete for "prometheus-adapter-auth-reader" RoleBinding
Starting delete for "rancher-monitoring-grafana-test" RoleBinding
Starting delete for "rancher-monitoring-grafana" RoleBinding
Starting delete for "rancher-monitoring-alertmanager" Role
Starting delete for "monitoring-config-edit" Role
Starting delete for "monitoring-config-view" Role
Starting delete for "rancher-monitoring-grafana-test" Role
Starting delete for "monitoring-dashboard-view" Role
Starting delete for "monitoring-dashboard-admin" Role
Starting delete for "monitoring-config-admin" Role
Starting delete for "monitoring-dashboard-edit" Role
Starting delete for "rancher-monitoring-grafana" Role
Starting delete for "prometheus-adapter-psp" ClusterRoleBinding
Starting delete for "psp-rancher-monitoring-prometheus-node-exporter" ClusterRoleBinding
Starting delete for "rancher-monitoring-kube-state-metrics" ClusterRoleBinding
Starting delete for "pushprox-kube-controller-manager-client" ClusterRoleBinding
Starting delete for "psp-rancher-monitoring-kube-state-metrics" ClusterRoleBinding
Starting delete for "pushprox-kube-controller-manager-proxy" ClusterRoleBinding
Starting delete for "prometheus-adapter-system-auth-delegator" ClusterRoleBinding
Starting delete for "pushprox-kube-etcd-client" ClusterRoleBinding
Starting delete for "prometheus-adapter-hpa-controller" ClusterRoleBinding
Starting delete for "pushprox-kube-etcd-proxy" ClusterRoleBinding
Starting delete for "pushprox-kube-proxy-client" ClusterRoleBinding
Starting delete for "pushprox-kube-proxy-proxy" ClusterRoleBinding
Starting delete for "pushprox-kube-scheduler-client" ClusterRoleBinding
Starting delete for "pushprox-kube-scheduler-proxy" ClusterRoleBinding
Starting delete for "rancher-monitoring-operator" ClusterRoleBinding
Starting delete for "rancher-monitoring-patch-sa" ClusterRoleBinding
Starting delete for "rancher-monitoring-grafana-clusterrolebinding" ClusterRoleBinding
Starting delete for "rancher-monitoring-operator-psp" ClusterRoleBinding
Starting delete for "prometheus-adapter-resource-reader" ClusterRoleBinding
Starting delete for "rancher-monitoring-prometheus" ClusterRoleBinding
Starting delete for "rancher-monitoring-prometheus-psp" ClusterRoleBinding
Starting delete for "rancher-monitoring-patch-sa" ClusterRole
Starting delete for "rancher-monitoring-grafana-clusterrole" ClusterRole
Starting delete for "psp-rancher-monitoring-kube-state-metrics" ClusterRole
Starting delete for "rancher-monitoring-kube-state-metrics" ClusterRole
Starting delete for "prometheus-adapter-resource-reader" ClusterRole
Starting delete for "prometheus-adapter-server-resources" ClusterRole
Starting delete for "prometheus-adapter-psp" ClusterRole
Starting delete for "psp-rancher-monitoring-prometheus-node-exporter" ClusterRole
Starting delete for "pushprox-kube-controller-manager-client" ClusterRole
Starting delete for "pushprox-kube-controller-manager-proxy" ClusterRole
Starting delete for "pushprox-kube-etcd-client" ClusterRole
Starting delete for "pushprox-kube-etcd-proxy" ClusterRole
Starting delete for "pushprox-kube-proxy-client" ClusterRole
Starting delete for "pushprox-kube-proxy-proxy" ClusterRole
Starting delete for "pushprox-kube-scheduler-client" ClusterRole
Starting delete for "pushprox-kube-scheduler-proxy" ClusterRole
Starting delete for "rancher-monitoring-operator" ClusterRole
Starting delete for "rancher-monitoring-operator-psp" ClusterRole
Starting delete for "rancher-monitoring-prometheus" ClusterRole
Starting delete for "rancher-monitoring-prometheus-psp" ClusterRole
Starting delete for "monitoring-admin" ClusterRole
Starting delete for "monitoring-edit" ClusterRole
Starting delete for "monitoring-view" ClusterRole
Starting delete for "monitoring-ui-view" ClusterRole
Starting delete for "rancher-default-dashboards-linux" ConfigMap
Starting delete for "rancher-monitoring-grafana-config-dashboards" ConfigMap
Starting delete for "rancher-monitoring-grafana" ConfigMap
Starting delete for "grafana-nginx-proxy-config" ConfigMap
Starting delete for "rancher-monitoring-grafana-test" ConfigMap
Starting delete for "rancher-monitoring-prometheus-adapter" ConfigMap
Starting delete for "rancher-monitoring-grafana-datasource" ConfigMap
Starting delete for "rancher-monitoring-apiserver" ConfigMap
Starting delete for "rancher-monitoring-cluster-total" ConfigMap
Starting delete for "rancher-monitoring-controller-manager" ConfigMap
Starting delete for "rancher-monitoring-etcd" ConfigMap
Starting delete for "rancher-monitoring-k8s-coredns" ConfigMap
Starting delete for "rancher-monitoring-k8s-resources-cluster" ConfigMap
Starting delete for "rancher-monitoring-k8s-resources-namespace" ConfigMap
Starting delete for "rancher-monitoring-k8s-resources-node" ConfigMap
Starting delete for "rancher-monitoring-k8s-resources-pod" ConfigMap
Starting delete for "rancher-monitoring-k8s-resources-workload" ConfigMap
Starting delete for "rancher-monitoring-k8s-resources-workloads-namespace" ConfigMap
Starting delete for "rancher-monitoring-kubelet" ConfigMap
Starting delete for "rancher-monitoring-namespace-by-pod" ConfigMap
Starting delete for "rancher-monitoring-namespace-by-workload" ConfigMap
Starting delete for "rancher-monitoring-node-cluster-rsrc-use" ConfigMap
Starting delete for "rancher-monitoring-node-rsrc-use" ConfigMap
Starting delete for "rancher-monitoring-nodes" ConfigMap
Starting delete for "rancher-monitoring-statefulset" ConfigMap
Starting delete for "rancher-monitoring-workload-total" ConfigMap
Starting delete for "rancher-monitoring-persistentvolumesusage" ConfigMap
Starting delete for "rancher-monitoring-pod-total" ConfigMap
Starting delete for "rancher-monitoring-ingress-nginx" ConfigMap
Starting delete for "rancher-monitoring-prometheus" ConfigMap
Starting delete for "rancher-default-dashboards-home" ConfigMap
Starting delete for "rancher-monitoring-proxy" ConfigMap
Starting delete for "rancher-default-dashboards-etcd" ConfigMap
Starting delete for "rancher-monitoring-scheduler" ConfigMap
Starting delete for "prometheus-nginx-proxy-config" ConfigMap
Starting delete for "rancher-default-dashboards-k8s" ConfigMap
Starting delete for "rancher-monitoring-grafana" Secret
Starting delete for "rancher-monitoring-patch-sa" ServiceAccount
Starting delete for "pushprox-kube-etcd-client" ServiceAccount
Starting delete for "rancher-monitoring-grafana" ServiceAccount
Starting delete for "rancher-monitoring-grafana-test" ServiceAccount
Starting delete for "rancher-monitoring-kube-state-metrics" ServiceAccount
Starting delete for "rancher-monitoring-prometheus-adapter" ServiceAccount
Starting delete for "rancher-monitoring-prometheus-node-exporter" ServiceAccount
Starting delete for "pushprox-kube-controller-manager-client" ServiceAccount
Starting delete for "pushprox-kube-controller-manager-proxy" ServiceAccount
Starting delete for "pushprox-kube-etcd-proxy" ServiceAccount
Starting delete for "pushprox-kube-scheduler-proxy" ServiceAccount
Starting delete for "rancher-monitoring-alertmanager" ServiceAccount
Starting delete for "rancher-monitoring-operator" ServiceAccount
Starting delete for "rancher-monitoring-prometheus" ServiceAccount
Starting delete for "pushprox-kube-proxy-client" ServiceAccount
Starting delete for "pushprox-kube-proxy-proxy" ServiceAccount
Starting delete for "pushprox-kube-scheduler-client" ServiceAccount
Starting delete for "pushprox-kube-scheduler-proxy" PodSecurityPolicy
Starting delete for "pushprox-kube-etcd-client" PodSecurityPolicy
Starting delete for "pushprox-kube-etcd-proxy" PodSecurityPolicy
Starting delete for "pushprox-kube-proxy-client" PodSecurityPolicy
Starting delete for "pushprox-kube-controller-manager-proxy" PodSecurityPolicy
Starting delete for "rancher-monitoring-alertmanager" PodSecurityPolicy
Starting delete for "pushprox-kube-proxy-proxy" PodSecurityPolicy
Starting delete for "pushprox-kube-scheduler-client" PodSecurityPolicy
Starting delete for "rancher-monitoring-operator" PodSecurityPolicy
Starting delete for "rancher-monitoring-grafana-test" PodSecurityPolicy
Starting delete for "rancher-monitoring-prometheus" PodSecurityPolicy
Starting delete for "rancher-monitoring-patch-sa" PodSecurityPolicy
Starting delete for "rancher-monitoring-grafana" PodSecurityPolicy
Starting delete for "rancher-monitoring-kube-state-metrics" PodSecurityPolicy
Starting delete for "rancher-monitoring-prometheus-adapter" PodSecurityPolicy
Starting delete for "rancher-monitoring-prometheus-node-exporter" PodSecurityPolicy
Starting delete for "pushprox-kube-controller-manager-client" PodSecurityPolicy
Starting delete for "default-allow-all" NetworkPolicy
Starting delete for "default-allow-all" NetworkPolicy
Starting delete for "rancher-monitoring-alertmanager" Alertmanager
Starting delete for "rancher-monitoring-admission" MutatingWebhookConfiguration
Starting delete for "rancher-monitoring-prometheus" Prometheus
Starting delete for "rancher-monitoring-alertmanager.rules" PrometheusRule
Starting delete for "rancher-monitoring-kubernetes-apps" PrometheusRule
Starting delete for "rancher-monitoring-etcd" PrometheusRule
Starting delete for "rancher-monitoring-general.rules" PrometheusRule
Starting delete for "rancher-monitoring-k8s.rules" PrometheusRule
Starting delete for "rancher-monitoring-kube-apiserver-availability.rules" PrometheusRule
Starting delete for "rancher-monitoring-kube-apiserver-slos" PrometheusRule
Starting delete for "rancher-monitoring-kube-apiserver.rules" PrometheusRule
Starting delete for "rancher-monitoring-kube-prometheus-general.rules" PrometheusRule
Starting delete for "rancher-monitoring-kube-prometheus-node-recording.rules" PrometheusRule
Starting delete for "rancher-monitoring-prometheus" PrometheusRule
Starting delete for "rancher-monitoring-kubernetes-resources" PrometheusRule
Starting delete for "rancher-monitoring-kubernetes-storage" PrometheusRule
Starting delete for "rancher-monitoring-kubernetes-system-apiserver" PrometheusRule
Starting delete for "rancher-monitoring-kubernetes-system-controller-manager" PrometheusRule
Starting delete for "rancher-monitoring-kubernetes-system-kubelet" PrometheusRule
Starting delete for "rancher-monitoring-kubernetes-system-scheduler" PrometheusRule
Starting delete for "rancher-monitoring-kubernetes-system" PrometheusRule
Starting delete for "rancher-monitoring-node-exporter.rules" PrometheusRule
Starting delete for "rancher-monitoring-node-exporter" PrometheusRule
Starting delete for "rancher-monitoring-node-network" PrometheusRule
Starting delete for "rancher-monitoring-node.rules" PrometheusRule
Starting delete for "rancher-monitoring-prometheus-operator" PrometheusRule
Starting delete for "rancher-monitoring-kube-state-metrics" PrometheusRule
Starting delete for "rancher-monitoring-kube-scheduler.rules" PrometheusRule
Starting delete for "rancher-monitoring-kubelet.rules" PrometheusRule
Starting delete for "rancher-monitoring-kube-proxy" ServiceMonitor
Starting delete for "rancher-monitoring-kube-controller-manager" ServiceMonitor
Starting delete for "rancher-monitoring-kube-etcd" ServiceMonitor
Starting delete for "rancher-monitoring-ingress-nginx" ServiceMonitor
Starting delete for "rancher-monitoring-alertmanager" ServiceMonitor
Starting delete for "rancher-monitoring-coredns" ServiceMonitor
Starting delete for "rancher-monitoring-apiserver" ServiceMonitor
Starting delete for "rancher-monitoring-kube-state-metrics" ServiceMonitor
Starting delete for "rancher-monitoring-kubelet" ServiceMonitor
Starting delete for "rancher-monitoring-kube-scheduler" ServiceMonitor
Starting delete for "rancher-monitoring-operator" ServiceMonitor
Starting delete for "rancher-monitoring-prometheus" ServiceMonitor
Starting delete for "rancher-monitoring-node-exporter" ServiceMonitor
Starting delete for "rancher-monitoring-grafana" ServiceMonitor
Starting delete for "rancher-monitoring-admission" ValidatingWebhookConfiguration
manifest-0

release "rancher-monitoring" uninstalled

---------------------------------------------------------------------
SUCCESS: helm uninstall --namespace=cattle-monitoring-system rancher-monitoring
---------------------------------------------------------------------
```

**Comment by thedadams** [comment]
@sowmyav27 Are istio and rancher-monitoring the only two for 2.5.8?

**Comment by sowmyav27** [comment]
Closing this issue as validations for Istio and monitoring are done. Separate issue has been logged to track changes for CIS, OPA, Logging v2 charts.

**Comment by FrolovVyacheslav** [comment]
I am using a rancher installed on single RKE cluster with:
RKE version 1.2.13
Kubectl version 1.21.6
Helm version 3.7.1
helm install rancher rancher-stable/rancher --namespace cattle-system --set hostname=examplecom --set ingress.tls.source=letsEncrypt --set letsEncrypt.email=example.mail.com --version 2.5.10`
In rancher-webhook pod logs I got the same errors:
```time="2021-12-10T21:17:36Z" level=info msg="Rancher-webhook version dev (HEAD) is starting"
time="2021-12-10T21:17:36Z" level=info msg="Listening on :9443"
time="2021-12-10T21:17:36Z" level=info msg="Creating new TLS secret for cattle-webhook-tls (count: 1): map[listener.cattle.io/cn-rancher-webhook.cattle-system.svc:rancher-webhook.cattle-system.svc listener.cattle.io/hash:7aaa4f3db00be92fc19d8a277022303e8c09763c46838b928cb6b5566a34ae31]"
time="2021-12-10T21:17:36Z" level=info msg="Active TLS secret cattle-webhook-tls (ver=5249) (count 1): map[listener.cattle.io/cn-rancher-webhook.cattle-system.svc:rancher-webhook.cattle-system.svc listener.cattle.io/hash:7aaa4f3db00be92fc19d8a277022303e8c09763c46838b928cb6b5566a34ae31]"
W1210 21:17:37.863094       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1210 21:17:38.251784       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
time="2021-12-10T21:17:38Z" level=info msg="Starting rbac.authorization.k8s.io/v1, Kind=ClusterRole controller"
time="2021-12-10T21:17:38Z" level=info msg="Starting rbac.authorization.k8s.io/v1, Kind=ClusterRoleBinding controller"
time="2021-12-10T21:17:38Z" level=info msg="Starting apiregistration.k8s.io/v1, Kind=APIService controller"
time="2021-12-10T21:17:38Z" level=info msg="Starting rbac.authorization.k8s.io/v1, Kind=Role controller"
time="2021-12-10T21:17:38Z" level=info msg="Starting management.cattle.io/v3, Kind=RoleTemplate controller"
time="2021-12-10T21:17:38Z" level=info msg="Starting management.cattle.io/v3, Kind=GlobalRole controller"
time="2021-12-10T21:17:38Z" level=info msg="Starting /v1, Kind=Secret controller"
time="2021-12-10T21:17:38Z" level=info msg="Sleeping for 15 seconds then applying webhook config"
time="2021-12-10T21:17:38Z" level=info msg="Starting apiextensions.k8s.io/v1beta1, Kind=CustomResourceDefinition controller"
time="2021-12-10T21:17:38Z" level=info msg="Starting rbac.authorization.k8s.io/v1, Kind=RoleBinding controller"
time="2021-12-10T21:17:38Z" level=info msg="Active TLS secret cattle-webhook-tls (ver=5249) (count 1): map[listener.cattle.io/cn-rancher-webhook.cattle-system.svc:rancher-webhook.cattle-system.svc listener.cattle.io/hash:7aaa4f3db00be92fc19d8a277022303e8c09763c46838b928cb6b5566a34ae31]"
E1210 21:18:37.498032       1 gvks.go:69] failed to sync schemas: unable to retrieve the complete list of server APIs: fleet.cattle.io/v1alpha1: the server could not find the requested resource, gitjob.cattle.io/v1: the server could not find the requested resource
E1210 21:18:38.419409       1 gvks.go:69] failed to sync schemas: unable to retrieve the complete list of server APIs: gitjob.cattle.io/v1: the server could not find the requested resource
E1210 21:18:39.553534       1 gvks.go:69] failed to sync schemas: unable to retrieve the complete list of server APIs: gitjob.cattle.io/v1: the server could not find the requested resource
E1210 21:18:40.400385       1 gvks.go:69] failed to sync schemas: unable to retrieve the complete list of server APIs: gitjob.cattle.io/v1: the server could not find the requested resource
E1210 21:18:41.317236       1 gvks.go:69] failed to sync schemas: unable to retrieve the complete list of server APIs: gitjob.cattle.io/v1: the server could not find the requested resource
E1210 21:18:42.118025       1 gvks.go:69] failed to sync schemas: unable to retrieve the complete list of server APIs: gitjob.cattle.io/v1: the server could not find the requested resource
E1210 21:18:42.430836       1 gvks.go:69] failed to sync schemas: unable to retrieve the complete list of server APIs: gitjob.cattle.io/v1: the server could not find the requested resource
W1210 21:23:07.262189       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1210 21:30:41.264276       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1210 21:36:46.271071       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1210 21:42:47.277879       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1210 21:50:45.282625       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1210 21:58:24.287485       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1210 22:04:30.296934       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1210 22:13:09.307550       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1210 22:22:55.314593       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1210 22:32:50.317819       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1210 22:38:46.322529       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1210 22:46:31.331039       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1210 22:53:33.337317       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1210 23:01:33.364021       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1210 23:10:01.364925       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1210 23:16:10.369146       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1210 23:24:01.375481       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1210 23:32:55.381558       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1210 23:38:42.387797       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1210 23:46:19.396451       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1210 23:53:03.401679       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1210 23:59:42.405327       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 00:08:16.415481       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 00:14:49.419347       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 00:22:18.423904       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 00:27:28.428441       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 00:37:21.432644       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 00:42:50.437043       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 00:49:29.454593       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 00:59:19.450080       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 01:04:23.458652       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 01:11:07.463945       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 01:16:09.469622       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 01:22:27.476849       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 01:29:31.479333       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 01:35:33.483267       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 01:45:18.487457       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 01:54:50.519923       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 02:03:35.525548       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 02:12:17.531312       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 02:17:49.536635       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 02:25:29.540378       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 02:30:30.550855       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 02:39:03.551849       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 02:48:51.556634       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 02:57:47.559292       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 03:04:01.569192       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 03:09:30.572713       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 03:16:11.581556       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 03:24:54.582133       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 03:31:50.590259       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 03:40:28.590709       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 03:48:37.593207       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 03:54:01.595945       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 04:02:58.599664       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 04:09:19.610080       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 04:15:20.613068       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 04:22:16.615500       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 04:29:28.621762       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 04:35:04.628724       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 04:42:25.632032       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 04:50:01.639438       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 04:58:13.649500       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 05:04:57.655433       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 05:10:46.662674       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 05:15:52.665109       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 05:23:03.672183       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 05:32:48.677381       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 05:42:37.683728       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 05:51:26.691959       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 05:59:31.698684       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 06:07:59.702218       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 06:13:52.705432       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 06:20:55.708621       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 06:29:07.715115       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 06:38:46.720281       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 06:45:50.725305       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 06:54:56.753823       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 07:04:14.745354       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 07:10:08.750296       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W1211 07:15:46.767235       1 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
time="2021-12-11T07:17:37Z" level=info msg="Sleeping for 15 seconds then applying webhook config"
time="2021-12-11T07:17:37Z" level=info msg="Active TLS secret cattle-webhook-tls (ver=5249) (count 1): map[listener.cattle.io/cn-rancher-webhook.cattle-system.svc:rancher-webhook.cattle-system.svc listener.cattle.io/hash:7aaa4f3db00be92fc19d8a277022303e8c09763c46838b928cb6b5566a34ae31]"```

**Comment by aiyengar2** [comment]
@FrolovVyacheslav I believe we shipped this in Rancher 2.6 here https://github.com/rancher/rancher/issues/33112#issuecomment-872582608

Not sure if we have backported it to the latest 2.5.x release line yet, but it's possible that this was fixed in 2.5.11.
